R/functions.R

```diff
@@ -127,39 +127,47 @@ summarizeAtlasExperiment <- function( experimentAccession, atlasExperimentDirect
 		stop( paste( "XML config file", atlasExperimentXMLfile, "does not exist or is not readable." ) )
 	}
     
-    cat( paste( "Reading experiment config from", atlasExperimentXMLfile, "...\n" ) )
+  cat( paste( "Reading experiment config from", atlasExperimentXMLfile, "...\n" ) )
```

Comment:
Could you fix the indentation levels please? The existing code is terrible for it as well, this is our chance to fix it!

Reply:
yes, we notices that yesterday